SwiftUIDev/SwiftUIDev/Spotify/Screens/LoginView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LoginView.swift
+//  SwiftUIDev
+//
+//  Created by LeeJiSoo on 2022/08/07.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    let buttonHorizontalPadding = 32.0
+    let mainLogoSize = 44.0
+    let signupButtonInterval = 10.0
+    let loginButtonPadding = 28.0
+    let titleInterval = 28.0
+    let titleAndButtonInterval = 40.0
+
+    var body: some View {
+        ZStack {
+            Image("spotify_login_image")
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .frame(minWidth: 0, maxWidth: .infinity)
+                .edgesIgnoringSafeArea(.all)
+            VStack {
+                Spacer()
+                title()
+                signupButtons()
+                    .padding(.horizontal, buttonHorizontalPadding)
+                loginButton()
+            }
+        }
+    }
+
+    private func signupButtons() -> some View {
+        VStack(spacing: signupButtonInterval) {
+            ForEach(SignupType.allCases, id: \.self) { type in
+                SignupButton(type: type)
+            }
+        }
+    }
+
+    private func loginButton() -> some View {
+        Text("Log in")
+            .loginFontStyle()
+            .padding(.vertical, loginButtonPadding)
+    }
+
+    private func title() ->  some View {
```

Comment:
p1;

앗..! 띄어쓰기가... (불편)
```suggestion
    private func title() -> some View {
```

Reply:
어머.. [ca07def](https://github.com/deslog/SwiftUIDev/pull/2/commits/ca07defb82693899e4ff86eba338cf2f7685d69e)